cmd/thanos/config.go

```diff
@@ -54,6 +55,10 @@ func (hc *httpConfig) registerFlag(cmd extkingpin.FlagClause) *httpConfig {
 	cmd.Flag("http-grace-period",
 		"Time to wait after an interrupt received for HTTP Server.").
 		Default("2m").SetValue(&hc.gracePeriod)
+	cmd.Flag(
```

Comment:
Sure. Will add that in!

Reply:
Currently, exporter-toolkit doesn't take in content(`byte[]`) of the TLS config file as an argument for its server, it only takes in the path(`string`) as highlighted [here](https://github.com/prometheus/exporter-toolkit/blob/master/web/tls_config.go#L178). So using `pathorcontent` isn't currently possible I think, since this flag fetches bytes. @bwplotka

Reply:
ack